pkg/insights/insightsuploader/insightsuploader.go

```diff
@@ -69,83 +69,70 @@ func New(summarizer Summarizer,
 	return ctrl
 }
 
-func (c *Controller) Run(ctx context.Context) {
+func (c *Controller) Run(ctx context.Context, initialDelay time.Duration) {
 	c.StatusController.UpdateStatus(controllerstatus.Summary{Healthy: true})
 
 	if c.client == nil {
 		klog.Infof("No reporting possible without a configured client")
 		return
 	}
 
-	// the controller periodically uploads results to the remote insights endpoint
 	cfg := c.configurator.Config()
-
 	interval := cfg.DataReporting.Interval
-	lastReported := c.reporter.LastReportedTime()
-	if !lastReported.IsZero() {
-		next := lastReported.Add(interval)
-		if now := time.Now(); next.After(now) {
-			c.initialDelay = wait.Jitter(next.Sub(now), 1.2)
-		}
-	}
-	klog.Infof("Reporting status periodically to %s every %s, starting in %s", cfg.DataReporting.UploadEndpoint, interval, c.initialDelay.Truncate(time.Second))
+	// set the initial upload delay as initial delay + 2 minutes
+	ud := 90 * time.Second
```

Comment:
Isn't two minutes (per the comment above) 120 seconds?

Reply:
:D correct. Thanks. I am going to change the comment to 1m30s :)